string directory
---
int64 status